ui/page/components/account_selector.go

```diff
@@ -76,17 +79,26 @@ func (as *AccountSelector) Handle() {
 				}
 				as.SetSelectedAccount(account)
 				as.callback(account)
-			}).Show()
+			}).
+			onModalExit(func() {
+				as.selectorModal = nil
+				as.ctxCancel()
```

Comment:
I don't know about stopping the tx+block notification listener when this modal is closed. It would still be needed to update the selected account when necessary. Also, if the modal were reopened, the listener isn't restarted. The listener should ideally only be canceled when the page holding the AccountSelector widget is closed.

Here's a thought: the widget should keep listening for potential balance changes as long as the page that is displaying it is open. To ensure that the listener is stopped when the page is no longer being displayed, add a `AccountSelector.WatchForBalanceChanges(ctx)` method takes a `ctx` parameter that will be supplied by the page that wants to display the selector widget. The method should start the listener. The page should cancel that ctx when the page is navigated from, which ensures that the listener is stopped.